content/_index.md

```diff
@@ -7,7 +7,7 @@ tags = []
 
 {{% section centered %}}
 
-# Launch Your Career in Tech:<br>Enroll in Our Free Training Program
+# Increasing Access to Careers in Tech
```

Comment:
Temporary changes like this should be conditional on the enroll site variable so they can be reverted by
changing the "enroll" variable in config.toml.

{{% if-equal enroll false %}}

temp content here

{{% /if-equal %}}